__tests__/app.test.js

```diff
@@ -3,6 +3,7 @@ const seed = require("../db/seeds/seed.js");
 const request = require("supertest");
 const app = require("../app.js");
 const testData = require("../db/data/test-data/index.js");
+require("jest-sorted");
```

Comment:
You shouldn't have to require this in - you may need to amend your package.json to set this up instead